config.js

```diff
@@ -29,3 +29,8 @@ cfg.mongoUrlTest = process.env.MONGO_URL_TEST || defaultMongoUrlTest;
 
 // Export configuration object
 module.exports = cfg;
+
+
+if (!cfg.twilioAccountSid || !cfg.twilioAuthToken) {
```

Comment:
I agree.
Code where write near the same time I read it @joliveros comment about `dotenv-safe` ;P